src/sass/_common/common.scss

```diff
@@ -389,8 +389,6 @@ button .barspinner.white {
 }
 
 .static_full {
-    margin-top: 40px;
-
```

Comment:
later we can remove `static_full` from pages that don't need it anymore. I didn't want this PR to get too big.